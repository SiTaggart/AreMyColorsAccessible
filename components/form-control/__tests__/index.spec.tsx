/* eslint-env jest */
/// <reference types="jest" />
import { expect } from '@jest/globals';

import { render } from '@testing-library/react';
import React from 'react';
import { FormControl } from '..';

describe('FormControl', (): void => {
  it('renders without crashing', (): void => {
    const { asFragment } = render(<FormControl>children</FormControl>);
    expect(asFragment()).toMatchSnapshot();
  });
});
