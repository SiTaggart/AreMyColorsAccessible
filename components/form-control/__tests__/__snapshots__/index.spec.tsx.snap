// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormControl renders without crashing 1`] = `
.emotion-0 {
  padding: 1rem;
  width: 100%;
}

@media (min-width:768px) {
  .emotion-0 {
    -webkit-flex: 1 1 auto;
    -ms-flex: 1 1 auto;
    flex: 1 1 auto;
    width: 50%;
  }
}

<div
  className="emotion-0 emotion-1"
>
  children
</div>
`;
